### PR TITLE
[Improvement]Add schema table backend_active_tasks

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes
```
mysql [information_schema]>desc information_schema.backend_active_tasks;
+---------------------------+--------------+------+-------+---------+-------+
| Field                     | Type         | Null | Key   | Default | Extra |
+---------------------------+--------------+------+-------+---------+-------+
| BE_ID                     | BIGINT       | Yes  | false | NULL    |       |
| FE_HOST                   | VARCHAR(256) | Yes  | false | NULL    |       |
| QUERY_ID                  | VARCHAR(256) | Yes  | false | NULL    |       |
| TASK_TIME_MS              | BIGINT       | Yes  | false | NULL    |       |
| TASK_CPU_TIME_MS          | BIGINT       | Yes  | false | NULL    |       |
| SCAN_ROWS                 | BIGINT       | Yes  | false | NULL    |       |
| SCAN_BYTES                | BIGINT       | Yes  | false | NULL    |       |
| BE_PEAK_MEMORY_BYTES      | BIGINT       | Yes  | false | NULL    |       |
| CURRENT_USED_MEMORY_BYTES | BIGINT       | Yes  | false | NULL    |       |
| SHUFFLE_SEND_BYTES        | BIGINT       | Yes  | false | NULL    |       |
| SHUFFLE_SEND_ROWS         | BIGINT       | Yes  | false | NULL    |       |
+---------------------------+--------------+------+-------+---------+-------+
11 rows in set (0.00 sec)
```

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: be/src/exec/schema_scanner/schema_backend_active_tasks.cpp [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+#include "runtime/runtime_query_statistics_mgr.h"
+#include "runtime/runtime_state.h"
+#include "vec/common/string_ref.h"
+#include "vec/core/block.h"
+#include "vec/data_types/data_type_factory.hpp"
+
+namespace doris {
+std::vector<SchemaScanner::ColumnDesc> SchemaBackendActiveTasksScanner::_s_tbls_columns = {
+        //   name,       type,          size
+        {"BE_ID", TYPE_BIGINT, sizeof(StringRef), false},
```
Comment (yiguolei) [inline be/src/exec/schema_scanner/schema_backend_active_tasks.cpp:30]: should be true to indicate it should be always nullable

**Review by yiguolei — DISMISSED** [review]

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by morningman — COMMENTED** [review]
Please update the document

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**